presto-spi/src/test/java/io/prestosql/spi/type/TestUnscaledDecimal128Arithmetic.java

```diff
@@ -768,4 +772,43 @@ private static Slice negate(Slice slice)
         }
         return ints;
     }
+
+    private static boolean isShort(BigInteger value)
```

Comment:
Do you you think it makes sense to extract refactors to separate preparatory commit before this one? That way it will be clear to see what kind of new tests case were added.

Reply:
I did think about it.
However those test will not work without the new methods and methods should not be commited without being properly tested. Refactor is strictly for the reason of testing new methods using existing test cases.

Exactly the same goes for assertRescale in later commit